expr/funcs.go

```diff
@@ -92,6 +92,7 @@ func init() {
 		"invert":                       {NewInvert, true},
 		"isNonNull":                    {NewIsNonNull, true},
 		"keepLastValue":                {NewKeepLastValue, true},
+		"linearRegression":             {NewLinearRegression, false},
```

Comment:
Why `false`?

Reply:
I figured we could leave it unstable for a while until we get a lot of confidence from real users. I'm confident the behavior is identical from my own testing with mtcmp though.

Reply:
No one will use it if it's unstable as they would need to send the process flag and you can't do that through grafana.